scale/job/configuration/interface/job_interface.py

```diff
@@ -10,26 +10,28 @@
 from jsonschema.exceptions import ValidationError
 
 from job.configuration.data.exceptions import InvalidData, InvalidConnection
-from job.configuration.interface import job_interface_1_2 as previous_interface
-from job.configuration.interface.exceptions import InvalidInterfaceDefinition, MissingSetting
+from job.configuration.interface import job_interface_1_3 as previous_interface
+from job.configuration.interface.exceptions import InvalidInterfaceDefinition
 from job.configuration.interface.scale_file import ScaleFileDescription
+from job.configuration.exceptions import MissingMount, MissingSetting
 from job.configuration.results.exceptions import InvalidResultsManifest
 from job.configuration.results.results_manifest.results_manifest import ResultsManifest
 from job.execution.container import SCALE_JOB_EXE_INPUT_PATH, SCALE_JOB_EXE_OUTPUT_PATH
 from scheduler.vault.manager import secrets_mgr
 
-
 logger = logging.getLogger(__name__)
 
-SCHEMA_VERSION = '1.3'
+SCHEMA_VERSION = '1.4'
+MODE_RO = 'ro'
+MODE_RW = 'rw'
 
 JOB_INTERFACE_SCHEMA = {
```

Comment:
Currently it won't fill in the default values for us: http://python-jsonschema.readthedocs.io/en/latest/faq/

If you want to pursue this, we can create an issue and tackle it later. It will take a little bit of work to validate and test.